.github/workflows/create-reports.yaml

```diff
@@ -0,0 +1,123 @@
+name: Automated Evaluation Reports and Projects 
```

Comment:
Let's replace **Evaluation Reports and Projects**  with **Qualification Reports**